Makefile

```diff
@@ -51,6 +51,10 @@ ephemeral: resources/public/admin
 ephemeral-prod: resources/public/admin
 	clojure -M:db-h2 -m lrsql.h2.main --persistent false
```

Comment:
Do we still need this `ephemeral-prod` target now that we have `ephemeral-oidc`?

Reply:
Makes sense - to be clear does that mean `ephemeral-prod` is useful outside of keycloak?

Reply:
Yes, I believe so. Specifically thinking here about cases where we're following up on a submitted issue or doing paid support.